src/Filament/Resources/LinkCheckResource/Pages/ListLinkChecks.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Portable\FilaCms\Filament\Resources\LinkCheckResource\Pages;
+
+use Filament\Actions;
+use Filament\Resources\Pages\ListRecords;
+use Portable\FilaCms\Filament\Resources\LinkCheckResource;
+use Illuminate\Support\Facades\Artisan;
+
+class ListLinkChecks extends ListRecords
+{
+    protected static string $resource = LinkCheckResource::class;
+
+    protected function getHeaderActions(): array
+    {
+        return [
+            Actions\Action::make('scan')
+                ->label('Check Links')
+                ->action(fn () => $this->executeScan())
+        ];
+    }
+
+    protected function executeScan()
+    {
+        Artisan::call('fila-cms:verify-links');
```

Comment:
I would make the Verify links a job that is queued and have this file and the console command calls.